x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_stream.tsx

```diff
@@ -111,150 +117,234 @@ export const PackagePolicyInputStreamConfig: React.FunctionComponent<{
     );
 
     return (
-      <EuiFlexGrid columns={2} id={isDefaultDatstream ? 'test123' : 'asas'}>
-        <ScrollAnchor ref={containerRef} />
-        <EuiFlexItem>
-          <EuiFlexGroup gutterSize="none" alignItems="flexStart">
-            <EuiFlexItem grow={1} />
-            <EuiFlexItem grow={5}>
-              <EuiSwitch
-                label={packageInputStream.title}
-                disabled={packagePolicyInputStream.keep_enabled}
-                checked={packagePolicyInputStream.enabled}
-                onChange={(e) => {
-                  const enabled = e.target.checked;
-                  updatePackagePolicyInputStream({
-                    enabled,
-                  });
-                }}
-              />
-              {packageInputStream.description ? (
-                <Fragment>
-                  <EuiSpacer size="s" />
-                  <EuiText size="s" color="subdued">
-                    <ReactMarkdown>{packageInputStream.description}</ReactMarkdown>
-                  </EuiText>
-                </Fragment>
-              ) : null}
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </EuiFlexItem>
-        <FlexItemWithMaxWidth>
-          <EuiFlexGroup direction="column" gutterSize="m">
-            {requiredVars.map((varDef) => {
-              if (!packagePolicyInputStream?.vars) return null;
-              const { name: varName, type: varType } = varDef;
-              const varConfigEntry = packagePolicyInputStream.vars?.[varName];
-              const value = varConfigEntry?.value;
-              const frozen = varConfigEntry?.frozen ?? false;
+      <>
```

Comment:
A lot of this diff is whitespace due to introducing a fragment at the top level. GitHub's ignore whitespace setting for this PR is probably your friend here 🙂